### ISSUE TITLE
version 2

### ISSUE THREAD

**Issue by matejthetree** [chatziko/FlashSocket.IO#3]
Any plans to update this lib to 2.0 socket io?

**Comment by chatziko** [comment]
I am currently using this in production with socket 2.0, it should work fine.

**Comment by matejthetree** [comment]
Yeah, need to pass websocket as transport though in opts. Pooling won't
work.

**Comment by chatziko** [comment]
I'm pretty sure polling works for me with socket.io 2.0.4.

Btw make sure you compile from source, the swc is no up-to-date.
